app/src/main/java/uc/edu/klopfsea/groupscheduler/ui/main/AddEvent.kt

```diff
@@ -100,7 +100,7 @@ class AddEvent : Activity(), AdapterView.OnItemSelectedListener {
         try{
             val formatter = SimpleDateFormat("dd/MM/yyyy", Locale.getDefault())
             val date = formatter.parse(dateString)
-            System.out.println(date)
+            println(date)
```

Comment:
Here is a very minor change that I'm sure was just something that may have gotten overlooked but in Kotlin you do not use system.out to print a line you can just simply leave it as println.  good to make sure you're coding Kotlin in 
Kotlin format and not java format.

Reply:
This appeared in a previous review as well.  When that occurs, first in wins.

Go one step further and use a log statement. No one will see a println, and println does not have the concept of priority, like logs do.